site/content/docs/user/quick-start.md

```diff
@@ -144,6 +144,9 @@ wait for 30 seconds, do `--wait 30s`, for 5 minutes do `--wait 5m`, etc.
 
 More usage can be discovered with `kind create cluster --help`.
 
+The kind can auto-detect the [docker], [podman], or [nerdctl] installed and choose the available one. If you want to turn off the auto-detect, use the environment variable `KIND_EXPERIMENTAL_PROVIDER=docker`, `KIND_EXPERIMENTAL_PROVIDER=podman` or `KIND_EXPERIMENTAL_PROVIDER=nerdctl` to
```

Comment:
@nunix is your concern addressed by this comment?

Reply:
@aojea yes all good on my side for the comment.

I will try again the auto-detect when the 0.23 is out maybe, as what I was trying to test was actually the auto-detect. But it's maybe on my side and with the comment, it provides the needed help.